code/datums/next_map.dm

```diff
@@ -80,10 +80,10 @@
 	path = "Island"
 	min_players = 25
 	
-/datum/next_map/line
-	name = "Frankenline Station"
-	path = "line"
-	min_players = 20
+#/datum/next_map/line
```

Comment:
This isn't how you comment in DM, use // for single line comments or /* (start) */ (stop) for multiline

Reply:
Noted for future use, thanks